Lesson34/gozde/lesson34-app/src/App.js

```diff
@@ -1,23 +1,72 @@
 import { List } from './components/List';
 import { Navbar } from './components/Navbar';
 import { PerformanceState } from './components/PerformanceState';
-import { useState } from 'react';
 import { todoData } from './data';
+import { UserProvider } from './UserContext';
 import './App.css';
+import { Account } from './components/Account';
+import { useContext, useState } from 'react';
+import { UserContext } from '../UserContext';
+
+
+
+// todo: create a new component that contains log in form, so that user can type username and password and click submit button to log in.
+// if user is logged in, show account component.
+//if user is logged out, show log in component.(need to use ternery operator for it in navbar)
+//create a form element to show in app page 
+
+
+export const LogIn = () => {
```

Comment:
Create a new folder in the 'components' folder, then move the LogIn component into it. It's a good practice to keep one component per file and this file is for App component.